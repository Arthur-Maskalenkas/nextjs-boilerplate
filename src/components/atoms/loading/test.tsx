import React from 'react'
import { render, RenderResult, screen } from '@testing-library/react'

import { Loading } from '.'

type SutTypes = {
	sut: RenderResult
}

const makeSut = (): SutTypes => {
  const sut = render(<Loading />)

  return {
    sut
  }
}

describe('Loading', () => {
  test('Should render Loading correctly', () => {
    const { sut: { container } } = makeSut()
    expect(container).toMatchSnapshot()

    expect(screen.getByLabelText('carregando')).toBeInTheDocument()
  })
})
