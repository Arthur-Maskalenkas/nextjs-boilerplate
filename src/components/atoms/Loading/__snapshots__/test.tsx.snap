// Jest Snapshot v1, https://goo.gl/fbAQLP

exports['Loading Should render Loading correctly 1'] = `
<div>
  <div
    class="container"
  >
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`
